src/components/AssetsDropZone/index.jsx

```diff
@@ -32,12 +32,18 @@ export default class AssetsDropZone extends React.Component {
     }
   }
 
-  setDropzonRef = (node) => {
-    this.dropzoneRef = node;
+  setDropZoneRef = (node) => {
```

Comment:
Bigger question - do we want them to all be inputs because they're like ref inputs, or should it be more specific to what is being ref'd, so with the Button one I thought the param name being button might be more clear, but didn't want to go through an change all up front. Happy to just use input if that's clear enough though.